### PR TITLE
Add command line option for '--min-days-unchanged' for reminder comments

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Comment by okurz** [comment]
@asdil12

File: openqa_review/openqa_review.py [hunk 1]
```diff
@@ -1069,7 +1072,7 @@ def reminder_comment_on_issue(ie, min_days_unchanged=14):
         issue.add_comment(comment)
 
 
-def reminder_comment_on_issues(report):
+def reminder_comment_on_issues(report, min_days_unchanged=14):
```
Comment (asdil12) [inline openqa_review/openqa_review.py]: Now the 14 is hardcoded twice.

Comment (asdil12) [inline openqa_review/openqa_review.py]: Actually it is even 3 times in the src now.

**Comment by okurz** [comment]
On Wednesday, 18 January 2017 07:27:39 CET Dominik Heidler wrote:
> asdil12 commented on this pull request.
> 
> > @@ -1069,7 +1072,7 @@ def reminder_comment_on_issue(ie, 
min_days_unchanged=14):
>          issue.add_comment(comment)
> 
> 
> -def reminder_comment_on_issues(report):
> +def reminder_comment_on_issues(report, min_days_unchanged=14):
> 
> Actually it is even 3 times in the src now.

I know, better extract the default value as a global constant?

**Comment by asdil12** [comment]
You nailed it :P

**Pull request merged**